### PR TITLE
v0.10 backports 2022-05-19

### PR DESCRIPTION
**Pull request by tklauser**
Two more backports from `master` wrt. the `cilium sysdump` command.

### CODE REVIEW

**Review by jibi — APPROVED** [review]

**Comment by tklauser** [comment]
No need to request reviews from teams for backport PRs. I've added an additional commit changing `CODEOWNERS` to only request top hat review.

**Pull request merged**
